x/metadata/types/oslocatorparams.go

```diff
@@ -0,0 +1,49 @@
+package types
```

Comment:
this file also should have a corresponding `_test.go` file

Reply:
![image](https://user-images.githubusercontent.com/1173553/111714059-6fb93980-8816-11eb-8582-735418e20d0c.png)
 added